BBKoffieTuin/Assets/Scripts/Route/RouteDebugHandler.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using TMPro;
+using UnityEngine;
+
+namespace Route
+{
+    /// <summary>
+    /// Route debug handler is a temporary class that will be used to debug the route system.
```

Comment:
temporary in develop??

Reply:
Yes het is nog steeds handig om te hebben voor nu (ook voor andere todo's) dus ik laat het er in.